src/identity.ts

```diff
@@ -0,0 +1,37 @@
+import { BeeDebug } from '@ethersphere/bee-js'
+import { createHash } from 'crypto'
+import { logger } from './logger'
+
+let interval: NodeJS.Timer | null
+let identity = 'unknown'
+
+export function getIdentity(): string {
```

Comment:
Can you please rename this function to indicate that it is not directly the identity but it is hash of it?

```suggestion
export function getHashedIdentity(): string {
```